libs/@local/hash-isomorphic-utils/src/flows/example-flows.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  inferUserEntitiesFromWebPageFlowDefinition,
+  researchTaskFlowDefinition,
+} from "./example-flow-definitions";
+import { triggerDefinitions } from "./step-definitions";
+import type { Flow } from "./types";
+
+export const researchTaskFlow: Flow = {
+  flowId: "researchTaskFlow",
+  trigger: {
+    definition: triggerDefinitions.userTrigger,
+  },
+  definition: researchTaskFlowDefinition,
```

Comment:
In reality a flow instance will be something like this, right?
```suggestion
  flowId: "abc123",
  flowDefinitionId: "researchTaskFlow"
```
and then the definition and trigger are looked up from the flow definition